src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  background-color: #fff;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  margin: 70px;
+  font-size: 16px;
+  line-height: 22px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+}
+
+.form-search-big {
+  padding-left: 62px;
+  height: 70px;
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat ;
+  background-position: 25px;
+  border: 0;
+}
+
+.form-search-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-search-big:focus {
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.form-search-small {
+  padding-left: 30px;
+  font-size: 14px;
+  height: 45px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat ;
```

Comment:
You repeated syles again. There are the same styles in .form-search-small and .form-search-big. Fix it, please. You can create a new class "form-search" and move there common styles

Reply:
Done!
thx for comments